.github/ISSUE_TEMPLATE/update_plugin.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Update plugin
+about: Request to update your plugin in the Marketplace
+title: "Update $REPOSITORY_NAME to $VERSION"
+labels: Plugin/Update
+assignees: hanzei
+---
+
+<!--
+Thank you very for continuing to develop and maintain your plugin. It will go through a review process to make sure all requirements are still met since the last release.
```

Comment:
```suggestion
Thank you very much for continuing to develop and maintain your plugin. It will go through a review process to make sure all requirements are still met since the last release.
```